presto-native-execution/presto_cpp/main/PrestoServer.cpp

```diff
@@ -1448,6 +1449,9 @@ void PrestoServer::registerFileReadersAndWriters() {
   if (SystemConfig::instance()->textWriterEnabled()) {
     velox::text::registerTextWriterFactory();
   }
+  if (SystemConfig::instance()->textReaderEnabled()) {
+    velox::text::registerTextReaderFactory();
+  }
```

Comment:
Just to double confirm, so internally we're registering text reader factory? @zacw7